ambry-api/src/main/java/com.github.ambry/router/RouterErrorCode.java

```diff
@@ -82,7 +82,11 @@
   /**
    * TTL of Blob has expired and so Blob cannot be retrieved.
    */
-  BlobExpired;
+  BlobExpired,
+  /**
+   * The range offsets provided for a getBlob operation go past the total size of a blob.
```

Comment:
This could also apply for other invalid scenarios, such as a blob not supporting range requests for whatever reason. Right now, it's only set when the range goes over the blob size.  I can change this comment to be a bit more general though.